src/library/browse/browsetablemodel.h

```diff
@@ -72,6 +72,13 @@ class BrowseTableModel final : public QStandardItemModel, public virtual TrackMo
     bool isColumnSortable(int column) override;
     TrackModel::SortColumnId sortColumnIdFromColumnIndex(int index) override;
     int columnIndexFromSortColumnId(TrackModel::SortColumnId sortColumn) override;
+    QString modelKey() const override {
+        return QString("0x%1/%2#%3")
+                .arg((quintptr)this, QT_POINTER_SIZE * 2, 16, QChar('0'))
```

Comment:
More C-style casts, redundant code and no QStringBuilder. I don't repeat review comments multiple times, please always fix all occurrences.

Reply:
The purpose of reviews is **not** to amend every single line of code and to dictate you what to do. The best feedback you could get from a review are just questions.

I spend the time for a thorough review. Please spend your time to read the feedback carefully to get the most out of it. Try to understand what a reviewer was thinking, consider the suggestions and come up with an even solution that would have prevented all questions, misunderstandings, and concerns in the first place.

Reply:
must have sliped through, thought I changed all modelKey() methodes